test/lib/test-init.ts

```diff
@@ -1,19 +1,24 @@
 import { ChildProcess, fork } from 'child_process';
-export const testListenPort = 1337;
-export const testLocalServer = `http://localhost:${testListenPort}`;
+import { boolVar } from '@balena/env-parsing';
+import type { PineTestOptions } from './pine-init';
+export const listenPortDefault = 1337;
+export const testLocalServer = `http://localhost:${listenPortDefault}`;
 
+console.log(`listenPortDefault:${JSON.stringify(listenPortDefault, null, 2)}`);
 export async function testInit(
-	fixturePath: string,
-	deleteDb: boolean = false,
-	pineListenPort: number = testListenPort,
+	options: Partial<PineTestOptions>,
 ): Promise<ChildProcess> {
 	try {
-		const processArgs = {
-			fixturePath,
-			testListenPort: pineListenPort,
-			deleteDb: deleteDb || process.env.DELETE_DB,
+		// manually setting defaults
+		if (options.configPath === undefined) {
+			throw new Error(`config path not set`);
+		}
```

Comment:
If you use
```ts
options: RequiredField<Partial<PineTestOptions>, 'configPath'>,
```
above then you can remove the need for this check (or at least make it able to be caught by typescript)
```suggestion
```
You can also use
```ts
OptionalField<PineTestOptions, 'listenPort' | 'deleteDb'>
```
which might be more accurate to your intentions